nvtabular/ops.py

```diff
@@ -788,6 +790,46 @@ def apply_mean_std(self, gdf, stats_context, cont_names):
         return new_gdf
 
 
+class NormalizeMinMax(DFOperator):
+    """
+    Standardizing the features around 0 with a standard deviation
+    of 1 is a common technique to compare measurements that have
+    different units. This operation can be added to the workflow
+    to standardize the features.
+
+    It performs Normalization using the min max method.
+
+    Although you can directly call methods of this class to
+    transform your continuous features, it's typically used within a
+    Workflow class.
+    """
+
+    default_in = CONT
+    default_out = CONT
+
+    @property
+    def req_stats(self):
+        return [MinMax()]
+
+    @annotate("NormalizeMinMax_op", color="darkgreen", domain="nvt_python")
+    def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None):
+        cont_names = target_columns
+        if not cont_names or not stats_context["mins"]:
+            return
+        gdf = self.apply_min_max(gdf, stats_context, cont_names)
+        return gdf
+
+    def apply_min_max(self, gdf, stats_context, cont_names):
+        new_gdf = cudf.DataFrame()
+        for name in cont_names:
+            dif = stats_context["maxs"][name] - stats_context["mins"][name]
+            if dif > 0:
+                new_col = f"{name}_{self._id}"
+                new_gdf[new_col] = (gdf[name] - stats_context["mins"][name]) / (dif)
+                new_gdf[new_col] = new_gdf[new_col].astype("float32")
```

Comment:
How are we going to handle a column that only contains a constant value? I think we still want to normalize it to be between 0 and 1, and include in the output.

How about if dif == 0 we just set to a constant value in [0,1]  - maybe 0.5?